src/modes/boot/boot.h

```diff
@@ -146,7 +146,8 @@ class BootMode : public vt_mode_manager::GameMode
     // ---------- Setup and refresh methods for boot menus
 
     //! \brief Returns whether at least one save is available to be loaded.
-    bool _SavesAvailable();
+    int32_t _NbSavesAvailable();
+    bool    _SavesAvailable();
```

Comment:
Note for self: I think we can get rid of _SaveAvailable() following your idea which is good, adapt the comment above this function to say it returns the number of available saves, and adapt the client logic.